AutoGVP/01-annotate_variants_CAVATICA_input.R

```diff
@@ -326,6 +327,42 @@ combined_tab_with_vcf_intervar <- autopvs1_results %>%
 
     ## adjust variables based on given rules described in README
     final_call = ifelse((evidencePVS1 == 1 &
+      ((evidencePS >= 1) |
+        (evidencePM >= 2) |
+        (evidencePM == 1 & evidencePP == 1) |
+        (evidencePP >= 2)) &
+      ((evidenceBA1) == 1 |
+        (evidenceBS >= 2) |
+        (evidenceBP >= 2) |
+        (evidenceBS >= 1 & evidenceBP >= 1) |
+        (evidenceBA1 == 1 & (evidenceBS >= 1 | evidenceBP >= 1)))), "Uncertain_significance",
```

Comment:
I don't think this criteria is adding anything unique, since evidenceBA1 == 1 is already specified above.